source/_docs/wp-config-php.md

```diff
@@ -138,4 +138,16 @@ The PHP 5.5 default is `&` and the PHP 5.3 default is `&amp;`.
 
 If the API expects `&` as an argument separator but receives `&amp;` (for example, when using http_build_query), you can override the default arg_separator.ouput value by adding the following line to `wp-config.php`:
 
-```ini_set('arg_separator.output', '&');```
+```php
+ini_set('arg_separator.output', '&');
+```
+
+### Adding actions or filters in `wp-config.php`
```

Comment:
Actions and filters shouldn't be added in `wp-config.php`

If they're added above the `require_once ABSPATH . 'wp-settings.php';` statement, then the site will fatal because `add_action()` and `add_filter()` won't be defined. If they're added below the `require_once ABSPATH . 'wp-settings.php';` statement, then the _entirety of WordPress has already been loaded_ and the actions / filters won't be applied.

Actions / filters should be added in a mu-plugin if they need to be run early. I think this particular proposed change only obscures the true issue.